scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -0,0 +1,579 @@
+ *                        http://proactive.inria.fr/team_members.htm
+ *  Contributor(s):
+ *
+ * ################################################################
+ * $$PROACTIVE_INITIAL_DEV$$
+ */
+package org.ow2.proactive.scheduler.authentication;
+
+import com.google.common.base.Charsets;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.TreeMultimap;
+import org.apache.commons.cli.*;
+import org.apache.commons.io.IOUtils;
+import org.objectweb.proactive.utils.SecurityManagerConfigurator;
+import org.ow2.proactive.authentication.FileLoginModule;
+import org.ow2.proactive.authentication.crypto.CreateCredentials;
+import org.ow2.proactive.authentication.crypto.Credentials;
+import org.ow2.proactive.authentication.crypto.HybridEncryptionUtil;
+import org.ow2.proactive.scheduler.core.properties.PASchedulerProperties;
+import org.ow2.proactive.utils.Tools;
+
+import java.io.*;
+import java.security.KeyException;
+import java.security.PublicKey;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.TreeMap;
+
+
+/**
+ * Manage user accounts in login.cfg and group.cfg files
+ *
+ * @author The ProActive Team
+ * @see org.ow2.proactive.authentication.crypto.Credentials
+ */
+public class ManageUsers {
+
+    private static final String newline = System.lineSeparator();
+
+    public static final String CREATE_OPTION = "C";
+    public static final String CREATE_OPTION_NAME = "create";
+    public static final String UPDATE_OPTION = "U";
+    public static final String UPDATE_OPTION_NAME = "update";
+    public static final String DELETE_OPTION = "D";
+    public static final String DELETE_OPTION_NAME = "delete";
+
+    public static final String HELP_OPTION = "h";
+    public static final String HELP_OPTION_NAME = "help";
+    public static final String LOGIN_OPTION = "l";
+    public static final String LOGIN_OPTION_NAME = "login";
+    public static final String PWD_OPTION = "p";
+    public static final String PWD_OPTION_NAME = "password";
+    public static final String GROUPS_OPTION = "g";
+    public static final String GROUPS_OPTION_NAME = "groups";
+    public static final String KEYFILE_OPTION = "kf";
+    public static final String KEYFILE_OPTION_NAME = "keyfile";
+    public static final String LOGINFILE_OPTION = "lf";
+    public static final String LOGINFILE_OPTION_NAME = "loginfile";
+    public static final String GROUPFILE_OPTION = "gf";
+    public static final String GROUPFILE_OPTION_NAME = "groupfile";
+
+
+    /**
+     * Entry point
+     *
+     * @param args arguments, try '-h' for help
+     * @throws IOException
+     * @throws ParseException
+     * @see org.ow2.proactive.authentication.crypto.Credentials
+     */
+    public static void main(String[] args) {
+
+        try {
+
+            manageUsers(args);
+
+        } catch (ManageUsersException e) {
+            System.err.println("ERROR : " + e.getMessage());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS106)